.meteor/release

```diff
@@ -1 +1 @@
-METEOR@1.8
+METEOR@1.8.0.2
```

Comment:
@impactmass did you intend to upgrade Meteor here?

Reply:
Yes. 

Here's a line from the PR description (with some more emphasis):
> The base image versioning is currently tied to Meteor versions, hence the new base image for 1.8.0.2

The versioning plan in the `reaction/base` repo is that we cut releases with meteor versions. So the binaries needed here are in the new version cut with the updated meteor. Hence this change.